tests/base_tests.py

```diff
@@ -149,7 +150,11 @@ def create_user_with_roles(username: str, roles: List[str]):
             db.session.commit()
             user_to_create = security_manager.find_user(username)
             assert user_to_create
-        user_to_create.roles = [security_manager.find_role(r) for r in roles]
+        user_to_create.roles = []
+        for chosen_user_role in roles:
+            if copy_roles:
+                security_manager.copy_role("Gamma", chosen_user_role, False)
```

Comment:
maybe.  now it is clearer?

Reply:
I think in a real use case, the roles users created will also not need to copy over all gamma permission. It will just contain some metadata about what user group this role represents (currently only configurable via role name). Copying permissions between roles should be avoided if possible since users can already have multiple roles.

@dpgaspar what do you think?